src/main/java/org/apache/pirk/responder/wideskies/ResponderProps.java

```diff
@@ -90,7 +127,7 @@ static boolean validateResponderProperties()
     }
 
     String platform = SystemConfiguration.getProperty(PLATFORM).toLowerCase();
-    if (!platform.equals("mapreduce") && !platform.equals("spark") && !platform.equals("standalone"))
+    if (!platform.equals("mapreduce") && !platform.equals("spark") && !platform.equals("storm") && !platform.equals("standalone"))
```

Comment:
Change this to a switch stmt using ENUM - the if list is only gonna get longer with addition of other backends and streaming and batch versions for each of them.